application/dash_application/utility/df_manipulation.py

```diff
@@ -136,32 +136,9 @@ def normalized_merged(df: DataFrame, merged_cols: List[str]):
 
 
 """
-Open a single instance of each cache, and use copies for the reports.
+qc-etl API keeps loaded caches in memory for fast reloading
```

Comment:
Out of curiosity, how much memory does this take at present?

Reply:
I am not sure. `flask` takes 1.2GB on my machine.